### ISSUE TITLE
softmax

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#70]
![](https://miro.medium.com/max/1400/1*670CdxchunD-yAuUWdI7Bw.png)
 Softmax is frequently appended to the last layer of an image classification network such as those in CNN ( VGG16 for example) used in ImageNet competitions.

![](https://miro.medium.com/max/1400/1*czt98NR7yXV6tdnJ0j7vBw.png)

**Comment by WatershedXiaolan** [comment]
Why use Softmax vs sigmoid?

Generally, we use softmax activation instead of sigmoid with the cross-entropy loss because softmax activation distributes the probability throughout each output node. But, since it is a binary classification, using sigmoid is same as softmax. For multi-class classification use sofmax with cross-entropy.Feb 18, 2018

**Comment by WatershedXiaolan** [comment]
why use softmax instead of max

The softmax function is differentiable everywhere, as well as having nice properties. For example, all the outputs are bounded between 0 and 1, and all the outputs sum to 1.
